### PR TITLE
code refactoring

### PR DESCRIPTION
**Pull request by nithin554**

### CODE REVIEW

File: src/main/java/com/backend/webapp/controller/LoginController.java [hunk 1]
```diff
@@ -37,7 +42,7 @@ public class LoginController {
     public ResponseEntity performLogin(@RequestBody LoginRequest loginRequest) {
         if (null == loginRequest.getEmail() || null == loginRequest.getPassword()) {
```
Comment (saikrishna-dev) [inline src/main/java/com/backend/webapp/controller/LoginController.java:43]:
```suggestion
        if (loginRequest.getEmail() == null || loginRequest.getPassword()  == null ) {
```
Computer science fundamentals xD

Comment (nithin554) [reply src/main/java/com/backend/webapp/controller/LoginController.java:43]: -_-

Comment (nithin554) [reply src/main/java/com/backend/webapp/controller/LoginController.java:43]: habit

Comment (saikrishna-dev) [reply src/main/java/com/backend/webapp/controller/LoginController.java:43]: correct it lol

**Review by saikrishna-dev — APPROVED** [review]

File: src/main/java/com/backend/webapp/controller/LoginController.java (hunk 1 above)
Comment (saikrishna-dev) [reply src/main/java/com/backend/webapp/controller/LoginController.java]: ???

Comment (nithin554) [reply src/main/java/com/backend/webapp/controller/LoginController.java]: check model classes

**Review by saikrishna-dev — APPROVED** [review]

**Pull request merged**